docs/articles/apex/refactoring.md

```diff
@@ -15,3 +15,25 @@ Validation of the new symbol name is performed before applying the rename. If th
 If the new name conflicts with an existing identifier name, we try to fully qualify the references to the existing name in contexts where the conflicts exist.
 
 ![GIF showing a renaming conflict](../../images/apex-rename-conflict.gif)
+
+## Refactor: Extract Constant
+
+You can extract valid literal expressions into constants. Literal expressions include: String, Integer, Long, Double, Decimal, and Boolean.
+
+This constant will be instantiated below the class declaration. If the expression is contained within an innerclass, the constant will be instantiated below the innerclass declaration. The new constant will always be defined with modifiers private, static, and final.
```

Comment:
"instantiated" is not the right word to use here. I would change it to something like "The new constant will be declared as a field in the containing class from where the refactoring was invoked". Alternately, you could just have an example here similar to what Jetbrains has: https://www.jetbrains.com/help/idea/extract-constant.html

Reply:
I like this. I will follow the example from Jetbrains. It will also mean that we are increasing the number of users that get value from the instructions (ease of access for people that cannot watch the video).